manual/de/ch_Tools_Assistants.xml

```diff
@@ -5144,107 +5157,40 @@
         </screenshot>
```

Comment:
Vor `</mediaobject>`:
`<caption><simpara>In diesem Bild werden alle verfügbaren Spalten angezeigt.</simpara></caption>`
oder so. Ist das Standard?

Reply:
Nö, kein Standard. Ich weiß allerdings auch nicht mehr, wie das normalerweise ist.